src/test/cypress/e2e/exercises/ExerciseImport.cy.ts

```diff
@@ -0,0 +1,180 @@
+        });
+    });
+
+    it('Imports modeling exercise', () => {
+        cy.login(instructor, `/course-management/${secondCourse.id}/exercises`);
+        courseManagementExercises.importModelingExercise();
+        courseManagementExercises.clickImportExercise(modelingExercise.id!);
+
+        checkField('#field_title', modelingExercise.title!);
+        checkField('#field_points', modelingExercise.maxPoints!);
+
+        modelingExerciseCreation.setReleaseDate(dayjs());
+        modelingExerciseCreation.setDueDate(dayjs().add(1, 'days'));
+        modelingExerciseCreation.setAssessmentDueDate(dayjs().add(2, 'days'));
+
+        modelingExerciseCreation.import().then((request: Interception) => {
+            const exercise = request.response!.body;
+            cy.login(studentOne, `/courses/${secondCourse.id}`);
+            courseOverview.startExercise(exercise.id!);
+            courseOverview.openRunningExercise(exercise.id!);
+            modelingExerciseEditor.addComponentToModel(exercise.id!, 1);
+            modelingExerciseEditor.addComponentToModel(exercise.id!, 2);
+            modelingExerciseEditor.addComponentToModel(exercise.id!, 3);
+            modelingExerciseEditor.submit().then((request: Interception) => {
+                expect(request.response!.body.submitted).to.be.true;
+                expect(request.response!.statusCode).to.eq(200);
+            });
+        });
+    });
+
+    it('Imports programming exercise', () => {
+        cy.login(instructor, `/course-management/${secondCourse.id}/exercises`);
+        courseManagementExercises.importProgrammingExercise();
+        courseManagementExercises.clickImportExercise(programmingExercise.id!);
+
+        checkField('#field_points', programmingExercise.maxPoints!);
+
+        programmingExerciseCreation.setTitle('Import Test');
+        programmingExerciseCreation.setShortName('importtest' + generateUUID());
+
+        programmingExerciseCreation.import().then((request: Interception) => {
+            const exercise = request.response!.body;
+            cy.login(studentOne, `/courses/${secondCourse.id}`);
+            courseOverview.startExercise(exercise.id!);
+            courseOverview.openRunningExercise(exercise.id!);
+            programmingExerciseEditor.makeSubmissionAndVerifyResults(exercise.id!, exercise.packageName!, partiallySuccessful, () => {
+                programmingExerciseEditor.getResultScore().contains(partiallySuccessful.expectedResult).and('be.visible');
+            });
+        });
+    });
+
+    after('Delete Course', () => {
+        cy.login(admin);
+        if (course.id) {
+            courseManagementRequest.deleteCourse(course.id).its('status').should('eq', 200);
+        }
+        if (secondCourse.id) {
+            courseManagementRequest.deleteCourse(secondCourse.id).its('status').should('eq', 200);
+        }
+    });
+});
+
+function checkField(field: string, value: any) {
+    cy.get(field).should('have.value', value);
+}
```

Comment:
Is that a general function that you should offer in a common place for all e2e tests?

Reply:
Good idea!